example/tests/unit/test_renderer_class_methods.py

```diff
@@ -9,12 +9,13 @@
 
 class ResourceSerializer(serializers.ModelSerializer):
     version = serializers.SerializerMethodField()
+    skipped_field = serializers.ReadOnlyField(source='foobar')
```

Comment:
This test currently only covers an error case I assume you have added to cover `except SkipField` line.

In your PR description you have outlined a case which would only fail if `include` is used and only works if this PR gets merged. Could you add such a test as well?

Reply:
Added